src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private Random random;
+    private ColorSupplier colorSupplier;
+
+    public Lottery() {
+        this.random = new Random();
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor().toString();
```

Comment:
As described in common mistakes, let's use name() method to get String representation of enum